browser/src/control/jsdialog/Widget.Combobox.js

```diff
@@ -183,6 +183,12 @@ JSDialog.combobox = function (parentContainer, data, builder) {
 	var container = L.DomUtil.create('div', 'ui-combobox ' + builder.options.cssClass, parentContainer);
 	container.id = data.id;
 
+	var labelText = L.Map.StateChangeHandler.prototype.Labels[data.command];
+	if (labelText) {
+		var label = L.DomUtil.create('label', '', container);
+		label.textContent = labelText;
```

Comment:
Why do we want that? combobox is the combobox field (input + dropdown).
If we want some label, we should put separate label widget next to it (in the .ui file in core, in the JSON).

Reply:
I did it, but for some reason, the parent container is removed and it messes up the combo box.
Somewhere it is not using well the combox which it is out of the scope of the task.

On the other hand, the label requires the LO Core .ui file changes that I am trying to avoid to do it.

Reply:
it's acctually what we should do (addin in .ui file).
Container can be replaced when we do some updates to the widget.
see `update` or `action` commands handling `executeAction` or `updateWidget` in JSDialogBuilder.
they work correctly as I know, but if you changed the structure here... action might not be able to change eg. text

Reply:
Ok, Please clarify in the task description that the changes also require changes in LibreOffice desktop accessibility.